<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>420</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 
                stop:0 #ffffff, stop:1 #d1d1d1); /* Glossy White to Silver */
    border: 2px solid #a0a0a0;
    border-radius: 8px;
    padding: 8px 15px;
    font-size: 14px;
    font-weight: bold;
    color: black;
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 
                stop:0 #f5f5f5, stop:1 #bdbdbd); /* Lighter when hovered */
    border: 2px solid #808080;
}

QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 
                stop:0 #d1d1d1, stop:1 #ffffff); /* Inverted Gloss Effect */
    border: 2px solid #606060;
}
</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qinput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: #ffffff;  /* White background */
    border: 2px solid #a0a0a0; /* Light grey border */
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 14px;
    color: black;
    selection-background-color: #c0c0c0; /* Light grey selection */
}

QLineEdit:focus {
    border: 2px solid #0078d7; /* Highlight with blue when focused */
    background: #f5f5f5; /* Slightly lighter background */
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #333333;  /* Dark grey text */
    font-size: 16px;
    font-weight: bold;
    background: transparent;  /* No background */
    padding: 5px;
    border: none;  /* No border for a clean look */
}
</string>
   </property>
   <property name="text">
    <string>Ajouter votre question ici :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="choice1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: #ffffff;  /* White background */
    border: 2px solid #a0a0a0; /* Light grey border */
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 14px;
    color: black;
    selection-background-color: #c0c0c0; /* Light grey selection */
}

QLineEdit:focus {
    border: 2px solid #0078d7; /* Highlight with blue when focused */
    background: #f5f5f5; /* Slightly lighter background */
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="choice2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: #ffffff;  /* White background */
    border: 2px solid #a0a0a0; /* Light grey border */
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 14px;
    color: black;
    selection-background-color: #c0c0c0; /* Light grey selection */
}

QLineEdit:focus {
    border: 2px solid #0078d7; /* Highlight with blue when focused */
    background: #f5f5f5; /* Slightly lighter background */
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>90</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #333333;  /* Dark grey text */
    font-size: 16px;
    font-weight: bold;
    background: transparent;  /* No background */
    padding: 5px;
    border: none;  /* No border for a clean look */
}
</string>
   </property>
   <property name="text">
    <string>1er choix :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #333333;  /* Dark grey text */
    font-size: 16px;
    font-weight: bold;
    background: transparent;  /* No background */
    padding: 5px;
    border: none;  /* No border for a clean look */
}
</string>
   </property>
   <property name="text">
    <string>2eme choix :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #333333;  /* Dark grey text */
    font-size: 16px;
    font-weight: bold;
    background: transparent;  /* No background */
    padding: 5px;
    border: none;  /* No border for a clean look */
}
</string>
   </property>
   <property name="text">
    <string>3eme choix :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #333333;  /* Dark grey text */
    font-size: 16px;
    font-weight: bold;
    background: transparent;  /* No background */
    padding: 5px;
    border: none;  /* No border for a clean look */
}
</string>
   </property>
   <property name="text">
    <string>4eme choix :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="choice3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: #ffffff;  /* White background */
    border: 2px solid #a0a0a0; /* Light grey border */
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 14px;
    color: black;
    selection-background-color: #c0c0c0; /* Light grey selection */
}

QLineEdit:focus {
    border: 2px solid #0078d7; /* Highlight with blue when focused */
    background: #f5f5f5; /* Slightly lighter background */
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="choice4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: #ffffff;  /* White background */
    border: 2px solid #a0a0a0; /* Light grey border */
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 14px;
    color: black;
    selection-background-color: #c0c0c0; /* Light grey selection */
}

QLineEdit:focus {
    border: 2px solid #0078d7; /* Highlight with blue when focused */
    background: #f5f5f5; /* Slightly lighter background */
}
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="rightanswer">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background: #ffffff;  /* White background */
    border: 2px solid #a0a0a0; /* Light grey border */
    border-radius: 6px;
    padding: 5px 10px;
    font-size: 14px;
    color: black;
}

QComboBox:hover {
    border: 2px solid #0078d7; /* Blue highlight on hover */
}

QComboBox:focus {
    border: 2px solid #0078d7; /* Blue border when focused */
    background: #f5f5f5; /* Light grey background */
}

QComboBox::drop-down {
    border: none;
    width: 25px;
    background: #f0f0f0;
    border-left: 1px solid #a0a0a0; /* Separate dropdown from main box */
    border-top-right-radius: 6px;
    border-bottom-right-radius: 6px;
}

QComboBox::down-arrow {
    image: url(down-arrow.png); /* Add a custom down arrow icon */
    width: 12px;
    height: 12px;
}
</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #333333;  /* Dark grey text */
    font-size: 16px;
    font-weight: bold;
    background: transparent;  /* No background */
    padding: 5px;
    border: none;  /* No border for a clean look */
}
</string>
   </property>
   <property name="text">
    <string>le choix correcte :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>490</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 
                stop:0 #ffffff, stop:1 #d1d1d1); /* Glossy White to Silver */
    border: 2px solid #a0a0a0;
    border-radius: 8px;
    padding: 8px 15px;
    font-size: 14px;
    font-weight: bold;
    color: black;
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 
                stop:0 #f5f5f5, stop:1 #bdbdbd); /* Lighter when hovered */
    border: 2px solid #808080;
}

QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 
                stop:0 #d1d1d1, stop:1 #ffffff); /* Inverted Gloss Effect */
    border: 2px solid #606060;
}
</string>
   </property>
   <property name="text">
    <string>Effacer</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>581</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
    background: #ffffff;  /* White background */
    border: 2px solid #a0a0a0; /* Light grey border */
    border-radius: 6px;
    padding: 5px;
    font-size: 14px;
    color: black;
    outline: none; /* Removes focus border */
}

QListWidget::item {
    padding: 8px;
    border-bottom: 1px solid #d0d0d0; /* Light separator between items */
}

QListWidget::item:selected {
    background: #0078d7;  /* Blue selection background */
    color: white;  /* White text for selected item */
    border-radius: 4px;
}

QListWidget::item:hover {
    background: #f0f0f0;  /* Light grey background on hover */
}

QListWidget::item:selected:active {
    background: #005bb5; /* Darker blue when clicked */
}
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
